### ISSUE TITLE
import/named does not report non-existent exports in yarn workspace

### ISSUE THREAD

**Issue by jd2rogers2** [benmosher/eslint-plugin-import#2005]
![Screen Shot 2021-03-12 at 9 45 07 AM](https://user-images.githubusercontent.com/16400333/110978069-c6ef7380-8317-11eb-89f5-b89f29afeb1f.png)

https://github.com/jd2rogers2/eslint-plugin-import-bug-minimal-repo for testing it for yourself

as seen above, no report for `import/named` despite not having an `helloFooV2` export from `foo/index.js`
`foo` and `bar` are both separate workspaces, `bar` lists `foo` as a dependency, and foo is in `node_modules`   
i would expect it to report
maybe a larger issue as there is also https://github.com/benmosher/eslint-plugin-import/issues/1832

**Comment by Skidle** [comment]
Hey @jd2rogers2 , I encountered a similar false positive error in my project and managed to fix it with the following:
- change exports in `foo` workspace to ES6 exports (same for imports in `bar`)
- add `'import/ignore': ['!^@my'],` under `settings` in eslint configuration. You're extending `airbnb-base` config, which has defined custom `import/ignore` settings: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/imports.js#L26. So you need to override those, because yarn workspaces are installed in `node_modules` (in your case under `@my` folder.

![image](https://user-images.githubusercontent.com/22978238/122053137-c4089100-cde6-11eb-894f-aeeb8ad1d98e.png)

**Comment by jd2rogers2** [comment]
@Skidle thanks for taking a look!
i'm not so sure it's because of the airbnb extension though
[here](https://github.com/jd2rogers2/eslint-plugin-import-bug-minimal-repo/pull/1) i remove the extension and just use the plugin and rule directly, and still `eslint .` does not report

**Comment by Skidle** [comment]
@jd2rogers2 yes, since my comment I realized that it happens with or withour `airbnb` plugin. If you don't use `airbnb` config, import/ignore will be `node_modules` by default, effectively ignoring yarn workspaces too.
So you need to add `'import/ignore': ['!^@my'],` to ignore anything that's not your workspace.

**Comment by jd2rogers2** [comment]
@Skidle still no luck on my end :(
i tried out your solution [here](https://github.com/jd2rogers2/eslint-plugin-import-bug-minimal-repo/pull/1/files) but i don't get any eslint reports for the import that i expect should receive an error
were you testing in your own repo or by cloning down mine? if you're working in another project would love to be able to find out the differences somehow
and thanks again for you help and time on this!

**Comment by Skidle** [comment]
@jd2rogers2 you need also to change `require` to `import from` in `bar.js`. Works for me [here](https://github.com/jd2rogers2/eslint-plugin-import-bug-minimal-repo/compare/master...Skidle:skidle-patch?expand=1) :)

**Comment by andrewzey** [comment]
I've just encountered this and was able to determine that the existence of the "import/resolver" key, even if the value is just an empty object, is what's causing `import/named` to not detect invalid imports.

Still working on figuring out how to get it working with an import/resolver, but that should give some clue.
